test/integration/test-handlers/echo-ts/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "echo-ts-handler",
+  "version": "1.0.0",
+  "description": "Sample Lambda echo handler with TypeScript for NodeJS",
+  "main": "app.js",
+  "author": "AWS Lambda",
+  "license": "Apache-2.0",
+  "scripts": {
+    "start": "node -r ts-node/register $(npm bin)/aws-lambda-ric index.handler"
```

Comment:
This is inconsistent with the `echo` project (using npx vs npm). Couldn't figure another way to load `ts-node`